device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
-    persist.data.qmi.adb_logmask=0
+    persist.data.qmi.adb_logmask=0 \
+    ro.adb.secure=0
+
+# for Gecko to support bluedroid stack
+PRODUCT_PACKAGES += \
+    bluetooth.default
+
+ENABLE_LIBRECOVERY := true
+
+# Enable virtual home button for b2g
+PRODUCT_PROPERTY_OVERRIDES += \
+    ro.moz.has_home_button=0 \
+    ro.moz.nfc.enabled=true \
+    ro.moz.ril.ipv6=true \
+    ro.moz.ril.signal_extra_int=true \
+
+PRODUCT_PACKAGES += \
+    nfcd \
+    init.sh
+
+PRODUCT_COPY_FILES += \
+    device/sony/shinano/qpnp_pon.kl:system/usr/keylayout/qpnp_pon.kl \
+    device/sony/shinano/volume.cfg:system/etc/volume.cfg
```

Comment:
I am not sure if we need timekeep; I think we can add it later in case of need.
- media_profiles.xml is included around line 61.
- Wifi hotspot tethering if functioning well.